pkg/proxy/iptables/proxier_test.go

```diff
@@ -3184,260 +3184,37 @@ func TestExternalTrafficPolicyCluster(t *testing.T) {
 	ipt := iptablestest.NewFake()
 	fp := NewFakeProxier(ipt)
 
-	expected := dedent.Dedent(`
-		*filter
-		:KUBE-NODEPORTS - [0:0]
-		:KUBE-SERVICES - [0:0]
```

Comment:
and squash the rest of this commit into the "implement "local-with-fallback" external traffic policy" commit.

(Also, remove `TestOnlyLocalNodePorts` and `onlyLocalNodePorts` too; they mistakenly got added as part of the merge conflict.)